### PR TITLE
refactor: improve onboarding feature flags

### PR DESCRIPTION
**Pull request by nicole-obrien**
## Summary
Please summarize your changes, describing __what__ they are and __why__ they were made.
Added more feature flags for all onboarding routes

### Changelog
```
- moved feature flags to folder
- separated onboarding flags
- added flags for all onboarding routes
- set flags for alpha
```

## Relevant Issues
Please list any related issues (e.g. bug, task).

## Type of Change
Please select any type below that is applicable to your changes, and delete those that are not.
- [ ] __Breaking__ - any change that would cause existing functionality to not work as expected
- [ ] __Chore__ - refactoring, build scripts or anything else that isn't user-facing
- [ ] __Docs__ - changes to the documentation
- [ ] __Fix__ - a change which fixes an issue
- [ ] __New__ - a change which implements a new feature
- [ ] __Update__ - a change which updates existing functionality

### Platforms
Please select any platforms where your changes have been tested.
- __Desktop__
	- [ ] MacOS
	- [ ] Linux
	- [ ] Windows
- __Mobile__
	- [ ] iOS
	- [ ] Android

### Instructions
Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

## Checklist
Please tick all of the following boxes that are relevant to your changes, and delete those that are not.
- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/features/features.js [hunk 1]
```diff
@@ -1,34 +1,7 @@
...
-            },
-            'private-net': {
-                enabled: false,
-            },
-        },
-    },
+import onboardingFeatures from './onboardingFeatures'
+
+const features = {
+    onboarding: onboardingFeatures,
```
Comment (MarkNerdi) [inline packages/shared/features/features.js:4]: if we extract the `onboardingFeatures` into an own file, i think we should do that also for the other

Comment (nicole-obrien) [reply packages/shared/features/features.js:4]: Yes will do, but limited the scope of this pr to onbaording

File: packages/shared/routes/setup/Create.svelte [hunk 2]
```diff
@@ -23,13 +24,31 @@
         <Text type="p" secondary classes="mb-8">{localize('views.create.body')}</Text>
     </div>
     <div slot="leftpane__action" class="flex flex-col space-y-4">
-        <Button icon="file" classes="w-full" secondary onClick={() => handleContinueClick(ProfileType.Software)}>
+        <Button
+            icon="file"
+            classes="w-full"
+            secondary
+            hidden={features?.onboarding?.[$newProfile?.networkProtocol]?.[$newProfile?.networkType]?.newProfile
+                ?.sofwareProfile?.hidden}
```
[the remark this answers is not in the document]
Comment (nicole-obrien) [reply packages/shared/routes/setup/Create.svelte:32]: yeah thats so the code is set up incase we want to easily hide it from feature flags

File: packages/shared/routes/setup/Create.svelte [hunk 3]
```diff
@@ -23,13 +24,31 @@
...
                 <Text type="p" secondary smaller>{localize('views.create.softwareAccount.description')}</Text>
             {/if}
         </Button>
-        <Button icon="chip" classes="w-full mb-8" secondary onClick={() => handleContinueClick(ProfileType.Ledger)}>
+        <Button
+            icon="chip"
+            classes="w-full mb-8"
+            secondary
+            hidden={features?.onboarding?.[$newProfile?.networkProtocol]?.[$newProfile?.networkType]?.newProfile
+                ?.ledgerProfile?.hidden}
```
[the remark this answers is not in the document]
Comment (nicole-obrien) [reply packages/shared/routes/setup/Create.svelte:47]: yeah thats so the code is set up incase we want to easily hide it from feature flags

**Pull request merged**